src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/model/request/VideoLinkBookingCreateRequest.kt

```diff
@@ -0,0 +1,94 @@
+package uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.model.request
+
+import com.fasterxml.jackson.annotation.JsonFormat
+import io.swagger.v3.oas.annotations.media.Schema
+import jakarta.validation.Valid
+import jakarta.validation.constraints.AssertTrue
+import jakarta.validation.constraints.Future
+import jakarta.validation.constraints.NotBlank
+import jakarta.validation.constraints.NotEmpty
+import jakarta.validation.constraints.NotNull
+import jakarta.validation.constraints.Size
+import java.net.URI
+import java.time.LocalDate
+import java.time.LocalTime
+
+data class VideoLinkBookingCreateRequest(
```

Comment:
Something learnt from previous project is to potentially isolate the model objects to single use, all objects related to the create request are in the one file.  The thinking hear being making it easier for changes going forwards.

There would/could be a trade off here as it could lead to potential duplication.